manifests/pod.yaml.j2

```diff
@@ -6,13 +6,15 @@ metadata:
   labels:
     tft-tests: "{{ index }}"
     tft-port: "{{ port }}"
+  annotations:
+    k8s.v1.cni.cncf.io/networks: {{ secondary_network_nad }}
```

Comment:
What if secondary_network_nad doesn't exist? Shouldn't this be optionally included?

Reply:
and maybe, `ConfConnection.secondary_network_nad` should be an `Optional[str]` to represent missing as `None` (instead of `""`).

Granted, we can interpret `""` as missing. To me, it feels cleaner to represent that as `None`.

Reply:
Thanks for the suggestion! Modified the PR to include `secondary_network_nad` optionally